README.md

```diff
@@ -1,23 +1,28 @@
-# balena Go Hello world
+# A Go server using net/http package
 
-This is a very simple project that is an example of how to run [Go][go] code on
-a [balena][balena-link] device.
+[![balena deploy button](https://www.balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/balena-io-examples/balena-go-hello-world)
 
-### balena Setup & Deployment
 
-To get this project up and running, you will need to signup for a balena account
-[here][signup-page] and set up an application and device. You'll find full
-details in our [Getting Started tutorial][gettingStarted-link].
+This is a simple skeleton [go][go] server project that works on any of the devices supported by [balena][balena-link].
+
+This project serves up a Welcome page on port `:80` of your balena device.
+
+To get this project up and running, you will need to signup for a balena account [here][signup-page] and set up a device, have a look at our [Getting Started tutorial][gettingStarted-link]. Once you are set up with balena, you will need to clone this repo locally or you can download this repository. 
+
+After downloading, navigate to the directory and `balena push` using the [balena CLI][balena-cli]. This command will package up and push the code to the balena builders, where it will be compiled and built and deployed to every device in the application fleet. When it completes, you'll have a node.js web server running on your device and see some logs on your [balenaCloud dashboard][balena-dashboard].
+
+```bash
+cd balena-python-hello-world/
```

Comment:
```suggestion
cd balena-go-hello-world/
```

Reply:
The PR's need some work, thanks for the review @20k-ultra